pkgs/development/compilers/osl/default.nix

```diff
@@ -52,15 +52,15 @@ in stdenv.mkDerivation rec {
 
   nativeBuildInputs = [
     bison
-    clang
+    clang_15
```

Comment:
not going going to block this PR, is there an upstream issue/PR to track clang 16 support ?

Reply:
I wasn't able to find one, but I could open one if that makes things simpler to track

Reply:
Actually, not sure how I missed it, but there is this: https://github.com/AcademySoftwareFoundation/OpenShadingLanguage/pull/1730
Unfortunately, applying the PR directly fails, but I guess it will be in one of the next releases.